### PR TITLE
enable S3 serverside encryption

### PR DESCRIPTION
**Pull request by nexusix**
S3 server-side encryption will be enabled by defining environment variable AWS_S3_SSE

### CODE REVIEW

**Comment by astub** [comment]
I need this. Any thing holding this back?

File: internal/awss3/storage.go [hunk 1]
```diff
@@ -227,14 +233,14 @@ func (s *Storage) PutChart(ctx context.Context, uri string, r io.Reader, chartMe
...
 		ctx,
 		&s3manager.UploadInput{
-			Bucket: aws.String(bucket),
-			Key:    aws.String(key),
-			ACL:    aws.String(acl),
-			Body:   r,
+			Bucket:               aws.String(bucket),
+			Key:                  aws.String(key),
+			ACL:                  aws.String(acl),
+			ServerSideEncryption: aws.String(os.Getenv(awsS3encryption)),
```
[the remark this answers is not in the document]
Comment (hypnoglow) [reply internal/awss3/storage.go]: probaby a helper function would be useful here:

```go
func getSSE() *string {
    sse := os.Getenv(awsS3encryption)
    if sse == "" {
        return nil
    }
    return &sse
}
```

File: README.md [hunk 2]
```diff
@@ -148,6 +148,10 @@ and others. To configure the plugin to work alternative S3 backend, just define
 
 See [these integration tests](https://github.com/hypnoglow/helm-s3/blob/master/hack/integration-tests-local.sh#L10) that use local minio docker container for a complete example.
 
+## Using S3 bucket ServerSide Encryption
+
+To enable S3 SSE export environment variable `AWS_S3_SSE` and set it to disired type for example `AES256`
```
Comment (hypnoglow) [inline README.md]: nits:
- please fix typo: disired -> desired
- add a dot in the end

**Comment by hypnoglow** [comment]
Thank you for the PR! Please note the issues I commented above.

**Comment by nexusix** [comment]
Fixed, thanks for review.

**Pull request merged**